tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-card.html

```diff
@@ -92,30 +98,15 @@
       :host[_expanded] {
         width: 100%;
       }
-      #chart-and-spinner-container {
+      #chart-container {
         display: flex;
         height: 300px;
         position: relative;
       }
-      :host[_expanded] #chart-and-spinner-container {
+      :host[_expanded] #chart-container {
         height: 600px;
       }
-      #line-chart-container {
-        width: 100%;
-        height: 100%;
-      }
-      #loading-spinner-container {
-        align-items: center;
-        bottom: 0;
-        display: flex;
-        justify-content: center;
-        left: 0;
-        pointer-events: none;
-        position: absolute;
-        right: 0;
-        top: 0;
-      }
-      vz-line-chart {
+      tf-line-chart-data-loader {
```

Comment:
Why `user-select: none`? (The scalars dashboard doesn't have this.)

Reply:
Ah! Thanks for the catch. `vz-line-chart` already prevents selection:
https://github.com/tensorflow/tensorboard/blob/534c23c6ab37b50dfb5f106b3d4433ce6323d040/tensorboard/components/vz_line_chart/vz-line-chart.html#L50

Removed.